### PR TITLE
dense inlet fix..

### PR DESCRIPTION
**Pull request by farrufat-cimne**
..and other fixes from spreaddem

### CODE REVIEW

**Comment by farrufat-cimne** [comment]
=========== DEM DISCONTINUUM TESTS ===========

==== TSUJI PAPER BENCHMARKS. SLIDING REGIME ==

DEM Benchmark 1: OK!........ Test 1 SUCCESSFUL (error: 0.4, time: 20.28s.)
DEM Benchmark 7: OK!........ Test 7 SUCCESSFUL (error: 0.0, time: 8.15s.)
DEM Benchmark 2: OK!........ Test 2 SUCCESSFUL (error: 0.13, time: 60.24s.)
DEM Benchmark 4: OK!........ Test 4 SUCCESSFUL (error: 4.63, time: 75.5s.)
DEM Benchmark 6: OK!........ Test 6 SUCCESSFUL (error: 3.38, time: 75.95s.)

==== THORNTON PAPER TESTS. FULL REGIME. HERTZIAN LAW ====

DEM Benchmark 11: OK!........ Test 11 (e=0.25) SUCCESSFUL
DEM Benchmark 11: OK!........ Test 11 (e=0.50) SUCCESSFUL
DEM Benchmark 11: OK!........ Test 11 (e=0.75) SUCCESSFUL
DEM Benchmark 8: OK!........ Test 8 SUCCESSFUL (error: 4.16, time: 73.15s.)
DEM Benchmark 3: OK!........ Test 3 SUCCESSFUL (error: 0.75, time: 102.39s.)
DEM Benchmark 11: OK!........ Test 11 (e=0.90) SUCCESSFUL
DEM Benchmark 5: OK!........ Test 5 SUCCESSFUL (error: 1.52, time: 108.32s.)

===== THORNTON PAPER TESTS. FULL REGIME. LINEAR LAW =====

DEM Benchmark 10: OK!........ Test 10 (e=0.25) SUCCESSFUL
DEM Benchmark 9: OK!........ Test 9 SUCCESSFUL (error: 0.97, time: 51.81s.)
DEM Benchmark 10: OK!........ Test 10 (e=0.50) SUCCESSFUL


==== WENSRICH PAPER TEST. ROLLING FRICTION ====

DEM Benchmark 12: OK!........ Test 12 SUCCESSFUL
DEM Benchmark 10: OK!........ Test 10 (e=0.75) SUCCESSFUL
DEM Benchmark 15: OK!........ Test 15 SUCCESSFUL


== BASIC CONTINUUM TESTS ==

DEM Benchmark 20: OK!........ Test 20 SUCCESSFUL
DEM Benchmark 21: OK!........ Test 21 SUCCESSFUL
DEM Benchmark 22: OK!........ Test 22 SUCCESSFUL
DEM Benchmark 14: OK!........ Test 14 SUCCESSFUL
DEM Benchmark 24: OK!........ Test 24 SUCCESSFUL
DEM Benchmark 23: OK!........ Test 23 SUCCESSFUL


===== DISCONTINUUM CLUSTERS TESTS =====

DEM Benchmark 30: OK!........ Test 30 SUCCESSFUL
DEM Benchmark 25: OK!........ Test 25 SUCCESSFUL
DEM Benchmark 31: OK!........ Test 31 SUCCESSFUL
DEM Benchmark 10: OK!........ Test 10 (e=0.90) SUCCESSFUL
DEM Benchmark 33: OK!........ Test 33 SUCCESSFUL
DEM Benchmark 32: OK!........ Test 32 SUCCESSFUL


======== DE/FE CONTACT BENCHMARKS ==========

DEM Benchmark 13: OK!........ Test 13 SUCCESSFUL
DEM Benchmark 16: OK!........ Test 16 SUCCESSFUL
DEM Benchmark 17: OK!........ Test 17 SUCCESSFUL

File: applications/DEM_application/custom_elements/spheric_particle.cpp [hunk 1]
```diff
@@ -1434,9 +1434,9 @@ void SphericParticle::ComputeAdditionalForces(array_1d<double, 3>& externally_ap
         const double vel_magnitude = DEM_MODULUS_3(vel);
         if (vel_magnitude != 0.0){
             const array_1d<double, 3> unitary_vel =  vel/vel_magnitude;
-            const double inlet_damping_coefficient = 1e4;
-            const array_1d<double, 3> damping_force = - inlet_damping_coefficient * GetMass() * vel;
-            const array_1d<double, 3> counter_force  = - 50.0 * gravity_force_magnitude * unitary_vel;
+            const double inlet_damping_coefficient = 1e3;
+            const array_1d<double, 3> damping_force = - inlet_damping_coefficient * GetMass() * vel_magnitude  * vel_magnitude * unitary_vel;
+            const array_1d<double, 3> counter_force  = -25.0 * gravity_force_magnitude * unitary_vel;
```
Comment (maceligueta) [inline applications/DEM_application/custom_elements/spheric_particle.cpp]: why is this force so big? 25 times the gravity? I would have used like 2 or 3...

Comment (farrufat-cimne) [reply applications/DEM_application/custom_elements/spheric_particle.cpp]: Only tested with 50 and 25. 25 looks fine.

**Comment by farrufat-cimne** [comment]
ping

**Review by maceligueta — APPROVED** [review]

**Pull request merged**